middle_end/flambda2/algorithms/patricia_tree.ml

```diff
@@ -22,39 +22,67 @@ open! Int_replace_polymorphic_compare
 (* The following is a "little endian" implementation. *)
 
 (* CR mshinwell: Can we fix the traversal order by swapping endianness? What
-   other (dis)advantages might that have? *)
+   other (dis)advantages might that have?
+
+   lmaurer: It would make [split] nearly as fast as [find]. One issue is we'd
+   want fast clz in order to implement [highest_bit]. *)
```

Comment:
That's ok though, because flambda-backend has fast clz.  Maybe turn this into a CR-someday, it might be good to fix the traversal order.